tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -89,6 +89,12 @@ class WskBasicUsageTests
         }
     }
 
+    it should "show help and usage info using the default language" in {
+        val env = Map("LANG" -> "de_DE")
+        // Call will fail with exit code 2 if language not supported
```

Comment:
@dubeejw - Sorry for the confusion.  I was referring to the comment that says the CLI will `fail with an exit code of 2 if language is not supported`.  In this scenario, the CLI should muddle on in English.